src/azure-cli/azure/cli/command_modules/batch/tests/latest/test_batch_mgmt_commands.py

```diff
@@ -54,7 +54,8 @@ def test_batch_general_arm_cmd(self, resource_group, storage_account):
             self.check('encryption.keySource', 'Microsoft.Batch'),
             self.check('resourceGroup', '{rg}')])
 
-        time.sleep(100)
+        if self.is_live or self.in_recording:
```

Comment:
Actually, the logic of `self.in_recording` already includes the logic of `self.is_live`.

Reply:
@wanghoppe Actually, please note that the logic of `self.in_recording` containing `self.is_live`, so it should be using `self.in_recording` ?

Reply:
@zhoxing-ms Thanks for reminding, changed it to `self.in_recording` instead